docs/samples/explanation/alibi/moviesentiment/kfserving_text_explainer.ipynb

```diff
@@ -0,0 +1,252 @@
+    "                                  limits={'cpu': '100m', 'memory': '2Gi'}))))\n",
+    "    \n",
+    "kfsvc = V1alpha2KFService(api_version=api_version,\n",
+    "                          kind=constants.KFSERVING_KIND,\n",
+    "                          metadata=client.V1ObjectMeta(\n",
+    "                              name='moviesentiment', namespace='kubeflow'),\n",
+    "                          spec=V1alpha2KFServiceSpec(default=default_endpoint_spec))\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Create KFService"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Call KFServingClient to create KFService."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "KFServing = KFServingClient()\n",
+    "KFServing.create(kfsvc)\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Check the KFService"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 60,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "NAME                 READY      DEFAULT_TRAFFIC CANARY_TRAFFIC  URL                                               \n",
+      "moviesentiment       True                   100                 http://moviesentiment-predict.kubeflow.example.com\n"
+     ]
+    }
+   ],
+   "source": [
+    "KFServing.get('moviesentiment', namespace='kubeflow', watch=True, timeout_seconds=120)\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Predict the sentiment for the sentence"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 61,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "34.67.11.82\n"
+     ]
+    }
+   ],
+   "source": [
+    "api_instance = kubernetes.client.CoreV1Api(kubernetes.client.ApiClient())\n",
+    "service = api_instance.read_namespaced_service(\"istio-ingressgateway\", \"istio-system\", exact='true')\n",
+    "cluster_ip = service.status.load_balancer.ingress[0].ip\n",
+    "print(cluster_ip)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 62,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "b'{\"predictions\": [0]}'\n",
+      "b'{\"predictions\": [1]}'\n"
+     ]
+    }
+   ],
+   "source": [
+    "url = \"http://\" + cluster_ip + \"/v1/models/moviesentiment:predict\"\n",
+    "headers = {'Host': 'moviesentiment-predict.kubeflow.example.com'}\n",
+    "response = requests.post(url, '{\"instances\":[\"This is a bad book .\"]}', headers=headers)\n",
+    "print(response.content)\n",
+    "response = requests.post(url, '{\"instances\":[\"This is a good book .\"]}', headers=headers)\n",
```

Comment:
The model is quite simple but we should probably try to find some more complex examples if possible to explain.